moveit_experimental/moveit_jog_arm/config/ur_simulated_config.yaml

```diff
@@ -7,9 +7,11 @@ use_gazebo: true # Whether the robot is started in a Gazebo simulation environme
 robot_link_command_frame:  world  # commands must be given in the frame of a robot link. Usually either the base or end effector
 command_in_type: "speed_units" # "unitless"> in the range [-1:1], as if from joystick. "speed_units"> cmds are in m/s and rad/s
 scale:
-  linear:  0.6  # Max linear velocity. Meters per publish_period. Units is [m/s]. Only used if command_in_type=="unitless"
-  rotational:  0.3 # Max angular velocity. Rads per publish_period. Units is [rad/s]. Only used if command_in_type=="unitless"
-  joint: 0.3  # Max joint angular/linear velocity. Rads or Meters per publish period. Units is [rad/s] or [m/s].
+  # Scale parameters are only used if command_in_type=="unitless"
+  linear:  0.6  # Max linear velocity. Meters per publish_period. Unit is [m/s]. Only used for Cartesian commands.
+  rotational:  0.3 # Max angular velocity. Rads per publish_period. Unit is [rad/s]. Only used for Cartesian commands.
+  # Max joint angular/linear velocity. Rads or Meters per publish period. Only used for joint commands on joint_command_in_topic.
```

Comment:
I'm now confused by this. I would expect this to be a velocity limit in rads or meters per second and not per publish period. Also, 10 is a really high example value here, no? Or does it count for all joints together...

Reply:
I wouldn't worry about it now, it's only used for "unitless" commands like from a joystick. They have to be scaled somehow.

Yeah, 10 is high. I'll change it to 1.